app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ReviewsController < ApplicationController
```

Comment:
Every PR needs to correspond to a new feature -- this PR doesn't add new functionality to the user (will always include some sort of front-end work to make use of the back-end changes). Please incorporate whatever feature these model changes are in service of